### DISCUSSION TITLE
You don't need CSS for that: All the ways you can use theme.json for styling

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#302] [Topic Ideas]
Some feedback from devs on a specific project (CC: @mikachan):

> They weren’t aware of all the options available in `theme.json` and didn’t realise how much SCSS could be moved there.

While most of the options are [documented in the Theme Handbook](https://developer.wordpress.org/themes/global-settings-and-styles/), those docs don't really show the various use cases. Sometimes, a walkthrough is needed for folks to understand how these pieces connect.

The goal is here to write a tutorial that's aimed at classic themers (and new themers) on really using `theme.json` to style their themes. The tutorial would cover things like:

- Defining presets (standard and custom) via the `settings` object
- Working with the `styles` object to design basic and more complex designs
- Using style variations (theme, block/section, color, and typography) instead of CSS
- When you must revert back to using CSS/stylesheets

Basically, we have the docs in place that describes **what** things are. Now it's time to show **how** to actually use them.

**Comment by bph** [comment]
This topic was approved in the 9/5 editorial group meeting. Please continue the conversation on the issue
- #308

**Comment by bph** [comment]
published: [Mastering theme.json: You might not need CSS](https://developer.wordpress.org/news/2024/10/17/mastering-theme-json-you-might-not-need-css/)
